data/filesdescription.yaml

```
@@ -0,0 +1,90 @@
+    that would be local to the webserver that they should not have access
+    to. Exploiting this type of attack can lead to the web application or
+    server being compromised.
+  REQUEST-931-APPLICATION-ATTACK-RFI.conf: >
+    These rules attempt to detect when a user is trying to include a
+    remote resource into the web application that will be executed.
+    Exploiting this type of attack can lead to the web application or
+    server being compromised.
+  REQUEST-932-APPLICATION-ATTACK-RCE.conf: TODO
+  REQUEST-933-APPLICATION-ATTACK-PHP.conf: TODO
+  REQUEST-934-APPLICATION-ATTACK-GENERIC.conf: TODO
+  REQUEST-941-APPLICATION-ATTACK-XSS.conf: TODO
+  REQUEST-942-APPLICATION-ATTACK-SQLI.conf: >
+    Within this configuration file we provide rules that protect against
+    SQL injection attacks. SQLi attackers occur when an attacker passes
+    crafted control characters to parameters to an area of the application
+    that is expecting only data. The application will then pass the
+    control characters to the database. This will end up changing the
+    meaning of the expected SQL query.
+  REQUEST-943-APPLICATION-ATTACK-SESSION-FIXATION.conf: >
+    These rules focus around providing protection against Session Fixation
+    attacks.
+  REQUEST-944-APPLICATION-ATTACK-JAVA.conf: TODO
+  REQUEST-949-BLOCKING-EVALUATION.conf: >
+    These rules provide the anomaly based blocking for a given request. If
+    you are in anomaly detection mode this file must not be deleted.
+  RESPONSE-950-DATA-LEAKAGES.conf:
+    These rules provide protection against data leakages that may occur
+    genericly
+  RESPONSE-951-DATA-LEAKAGES-SQL.conf: >
+    These rules provide protection against data leakages that may occur
+    from backend SQL servers. Often these are indicative of SQL injection
+    issues being present.
+  RESPONSE-952-DATA-LEAKAGES-JAVA.conf: >
+    These rules provide protection against data leakages that may occur
+    because of Java
+  RESPONSE-953-DATA-LEAKAGES-PHP.conf: >
+    These rules provide protection against data leakages that may occur
+    because of PHP
+  RESPONSE-954-DATA-LEAKAGES-IIS.conf: >
+    These rules provide protection against data leakages that may occur
+    because of Microsoft IIS.
+  RESPONSE-959-BLOCKING-EVALUATION.conf: >
+    These rules provide the anomaly based blocking for a given response.
+    If you are in anomaly detection mode this file must not be deleted.
+  RESPONSE-980-CORRELATION.conf: >
+    The rules in this configuration file facilitate the gathering of data
+    about successful and unsuccessful attacks on the server.
+  RESPONSE-999-EXCLUSION-RULES-AFTER-CRS.conf.example: >
+    This file is used to add LOCAL exceptions for your site. Often in this
+    file we would see rules that short-circuit inspection and allow
+    certain transactions to skip through inspection.
+  RESPONSE-980-CORRELATION.conf: TODO
+
+"3":
```

I would like this to be more specific, e.g.,:
```suggestion
versioned:
  "3":
```